testng-core/src/test/java/testhelper/JarCreator.java

```diff
@@ -1,4 +1,4 @@
-package org.testng.jarfileutils;
+package testhelper;
```

Comment:
What was the issue with the previous package?

Reply:
I found that some of the helpers resided here and some there. Just trying to unify all these helpers in one place so that they are easy to find ( I had to search through the code to find them 😄 )

Reply:
Feels wrong to not have all code under package org.testng.x
wouldn't it be better to but them directly under org.testng or org.testng.testhelper?

Reply:
@juherr fixed